src/main/java/ru/itmo/roguelike/render/RenderScheduler.java

```diff
@@ -5,6 +5,9 @@
 
 import java.util.TimerTask;
 
+/**
+ * Class which updates game states by time
```

Comment:
No. This class updates picture on screen, it calls `render()`, according to frequency setting